cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/network/CreateVpc.java

```diff
@@ -0,0 +1,170 @@
+     * You can specify the instance tenancy value for the VPC when you create it. You can't change this value for the VPC
+     * after you create it. For more information, see Dedicated Instances: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/dedicated-instance.html
+     * <p>
+     * Important: This is an idempotent operation. If you perform the operation more than once, Amazon EC2 doesn't return
+     * an error.
+     *
+     * @param endpoint                    Optional - Endpoint to which request will be sent.
+     *                                    Default: "https://ec2.amazonaws.com"
+     * @param identity                    ID of the secret access key associated with your Amazon AWS or IAM account.
+     *                                    Example: "AKIAIOSFODNN7EXAMPLE"
+     * @param credential                  Secret access key associated with your Amazon AWS or IAM account.
+     *                                    Example: "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
+     * @param proxyHost                   Optional - proxy server used to connect to Amazon API. If empty no proxy will
+     *                                    be used.
+     *                                    Default: ""
+     * @param proxyPort                   Optional - proxy server port. You must either specify values for both proxyHost
+     *                                    and proxyPort inputs or leave them both empty.
+     *                                    Default: ""
+     * @param proxyUsername               Optional - proxy server user name.
+     *                                    Default: ""
+     * @param proxyPassword               Optional - proxy server password associated with the proxyUsername input value.
+     *                                    Default: ""
+     * @param headers                     Optional - string containing the headers to use for the request separated by
+     *                                    new line (CRLF). The header name-value pair will be separated by ":".
+     *                                    Format: Conforming with HTTP standard for headers (RFC 2616)
+     *                                    Examples: "Accept:text/plain"
+     *                                    Default: ""
+     * @param queryParams                 Optional - string containing query parameters that will be appended to the URL.
+     *                                    The names and the values must not be URL encoded because if they are encoded
+     *                                    then a double encoded will occur. The separator between name-value pairs is "&"
+     *                                    symbol. The query name will be separated from query value by "=".
+     *                                    Examples: "parameterName1=parameterValue1&parameterName2=parameterValue2"
+     *                                    Default: ""
```

Comment:
empty defaults are redundant ... so they can be removed ( keeping the description short )